### PR TITLE
customGear - Custom M433 HEDP and migrating 40x56mm rounds to custom gear

### PR DESCRIPTION
**Pull request by lambdatiger**
This PR:
- Adds a 40x46mm M433 HEDP with appropriate penetrator and damage
- Moves the 40x53mm HV Mk19 rounds from `massCompat` to `customGear`

This PR requires https://github.com/BourbonWarfare/POTATO/pull/610

### CODE REVIEW

**Review by TheCandianVendingMachine — APPROVED** [review]
lgtm, this will be a nice addition. i love smaller AT weapons

File: addons/customGear/CfgAmmo.hpp [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        submunitionInitialOffset[] = {0, 0, -0.2};
+        submunitionInitSpeed = 1000;
+        submunitionParentSpeedCoef = 0;
+        triggerOnImpact = 1;
+    };
+
+    class AMMO(40x46mm_HEDP_M433_penetrator): ammo_Penetrator_Base {
+        hit = 70; // a guess
+        caliber = 4.2; // 63mm steel
+        submunitionConeType[] = {};
```
Comment (TheCandianVendingMachine) [inline addons/customGear/CfgAmmo.hpp:78]: we not overriding rhs spalling here?

**Review by TheCandianVendingMachine — CHANGES_REQUESTED** [review]

File: addons/customGear/CfgAmmo.hpp (hunk 1 above)
Comment (lambdatiger) [reply addons/customGear/CfgAmmo.hpp:78]: https://github.com/BourbonWarfare/POTATO/pull/587

Comment (TheCandianVendingMachine) [reply addons/customGear/CfgAmmo.hpp:78]: my question was more: why do we override spalling for the other pentrator and not this one

Comment (lambdatiger) [reply addons/customGear/CfgAmmo.hpp]: I understand now. That's old code that I have now removed.

**Review by TheCandianVendingMachine — APPROVED** [review]

**Review by PabstMirror — APPROVED** [review]

**Comment by lambdatiger** [comment]
Would you like me to handle this merge locally?

**Pull request merged**